src/main/java/com/airlines/entity/Airplane.java

```diff
@@ -0,0 +1,87 @@
+package com.airlines.entity;
+
+import java.time.LocalDate;
+
+public class Airplane {
+    private final int id;
+    private final String codeName;
+    private final String model;
```

Comment:
You have enum AirplaneModel and you should use it.
Otherwise you will be able write smth like that:
```
Airplane.builder()
    .model("KuKURUZNik")
    .build();
```
and this will be written into database which is not right.

But if you change it to 
`private final AirplaneModel model;`
it will be more safier.

Reply:
fixed in 8d230f9c3c3300d7da8f899a28eb4137f3bf8074